src/main/java/WayofTime/bloodmagic/item/ItemRitualDismantler.java

```diff
@@ -0,0 +1,81 @@
+import WayofTime.bloodmagic.ritual.RitualComponent;
+import WayofTime.bloodmagic.tile.TileMasterRitualStone;
+import WayofTime.bloodmagic.util.helper.NetworkHelper;
+import WayofTime.bloodmagic.util.helper.RitualHelper;
+import com.google.common.collect.Lists;
+import net.minecraft.block.Block;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.items.ItemHandlerHelper;
+
+import java.util.List;
+
+public class ItemRitualDismantler extends Item implements IVariantProvider {
+    public ItemRitualDismantler() {
+        setUnlocalizedName(BloodMagic.MODID + ".ritualDismantler");
+        setCreativeTab(BloodMagic.TAB_BM);
+        setMaxStackSize(1);
+    }
+
+    @Override
+    public EnumActionResult onItemUse(EntityPlayer player, World world, BlockPos pos, EnumHand hand, EnumFacing facing, float hitX, float hitY, float hitZ) {
+        return breakRitualStoneAtMasterStone(player, world, pos, player.getHeldItem(hand));
+    }
+
+    public EnumActionResult breakRitualStoneAtMasterStone(EntityPlayer player, World world, BlockPos pos, ItemStack stack) {
+        Block block = world.getBlockState(pos).getBlock();
+        TileEntity tileEntity = world.getTileEntity(pos);
+
+        if (tileEntity instanceof TileMasterRitualStone) {
+            TileMasterRitualStone masterRitualStone = (TileMasterRitualStone) tileEntity;
+            EnumFacing direction = masterRitualStone.getDirection();
+
+            String ritualName = RitualHelper.getValidRitual(world, pos);
+            masterRitualStone.setActive(false);
+
+            if (ritualName.equals("")) {
+                world.setBlockToAir(pos);
+                ItemHandlerHelper.giveItemToPlayer(player, new ItemStack(RegistrarBloodMagicBlocks.RITUAL_CONTROLLER));
+                NetworkHelper.getSoulNetwork(player).syphon(SoulTicket.item(stack, 250));
+                return EnumActionResult.SUCCESS;
+            }
+
+            Ritual ritual = BloodMagic.RITUAL_MANAGER.getRitual(ritualName);
+            List<RitualComponent> components = Lists.newArrayList();
+            ritual.gatherComponents(components::add);
+            for (RitualComponent component : components) {
+                BlockPos newPos = pos.add(component.getOffset(direction));
+                if (world.getBlockState(newPos).getBlock() instanceof BlockRitualStone) {
+                    world.setBlockToAir(newPos);
+                    ItemHandlerHelper.giveItemToPlayer(player, new ItemStack(RegistrarBloodMagicBlocks.RITUAL_STONE));
+                }
+            }
+
+            NetworkHelper.getSoulNetwork(player).syphon(SoulTicket.item(stack, 100));
```

Comment:
Why does it cost 250LP to destroy just the controller, but only 100LP to destroy the entire ritual?

Reply:
My thought process was that the MRS is more powerful. "The entire ritual" does not destroy the MRS.

For cases in which your MRS is above a pool of lava (RitualPump) and you really don't want to destroy the MRS with a pickaxe. Or just because.

In reality, most of the LP values are placeholders for more reasonable proposals.

Reply:
Variable cost up to a maximum for a whole ritual?